projects/ngrx.io/content/guide/migration/v13.md

```diff
@@ -97,6 +97,19 @@ StoreModule.forFeature(
 
 The `initialState` provided via the configuration object is now typed and needs to match the interface of the state interface.
 
+#### Testing: Reset mock store
+
+Mock stores are not reset automagically after each test. 
+
+When using Jasmine, reset the mock store after each test by adding the following to the `test.ts`:
+
+```ts
+import { MockStore } from '@ngrx/store/testing';
+
+jasmine.getEnv().afterEach(() => {
+  TestBed.inject(MockStore, null)?.resetSelectors();
+});
+```
```

Comment:
Let's also add a jest example:

```suggestion
```ts
import { MockStore } from '@ngrx/store/testing';

jasmine.getEnv().afterEach(() => {
  TestBed.inject(MockStore, null)?.resetSelectors();
});
```

When using Jest, reset the mock store after each test by using a `afterEach` hook:

```ts
import { MockStore } from '@ngrx/store/testing';

afterEach(() => {
  TestBed.inject(MockStore)?.resetSelectors();
});
```
```

Reply:
The suggestions are action weird, the following can be appended:

```ts
import { MockStore } from '@ngrx/store/testing';

afterEach(() => {
  TestBed.inject(MockStore)?.resetSelectors();
});
```